cdap-docs/developer-manual/source/advanced/configuring-resources.rst

```diff
@@ -84,7 +84,7 @@ in the ``default`` namespace, and would like to set the memory used by its YARN
       $ curl -w"\n" -X PUT "http://example.com:11015/v3/preferences" \
           -H 'Content-Type: application/json' -d '{ "task.mapper.system.resources.memory": 2048 }'
 
-- To set the memory used by the mapper of the *PurchaseHistoryBuilder* MapReduce job, you would use::
+- To set the memory used by the mapper of the *MyMapReduce* MapReduce job, you would use::
```

Comment:
is this correct? It seems the same as setting it for all MapReduce

Reply:
The example is setting this argument on just the one MapReduce job (line 96).